application/common/models/Password.php

```diff
@@ -280,4 +284,9 @@ public function markPwned(): void
             ]);
         }
     }
+
+    public static function hashPassword(string $password): string
+    {
+        return password_hash($password, self::HASH_ALGORITHM, ["cost" => self::HASH_COST]);
```

Comment:
Is this because the PHP Documentation says, "Note that this constant is designed to change over time as new and stronger algorithms are added to PHP."?

Reply:
Not necessarily, I just thought it seemed like a better option than "default".